app/javascript/components/explore/DifferentialExpressionPanel.jsx

```diff
@@ -166,7 +168,7 @@ function searchGenesFromTable(selectedGenes, searchGenes, logProps) {
 /** Table of DE data for genes */
 function DifferentialExpressionTable({
   genesToShow, searchGenes, clusterName, annotation, species, numRows,
-  bucketId, deFilePath, handleClear, isAuthorDe
+  bucketId, deFilePath, handleClear, isAuthorDe, facets
```

Comment:
I'm wondering if `facets` is potentially confusing here - we have plans to add actual search facets to many of our Explore tab plots this quarter, and this might get overloaded.